opsman.prolific

```diff
@@ -50,15 +196,15 @@ OpsManager is running and you can log in.
 - #pas-releng Slack
 
 ### Resources
-- https://docs.pivotal.io/pivotalcf/2-2/customizing/
+- https://docs.pivotal.io/pivotalcf/2-4/customizing/gcp.html
```

Comment:
Note: You can remove the version number from the URL to have this always redirect to the latest version, e.g.

```
https://docs.pivotal.io/pivotalcf/customizing/gcp.html
```